pubsub/google/cloud/pubsub_v1/publisher/batch/base.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import abc
+import collections
+
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class BaseBatch(object):
```

Comment:
since this module is named `base`, you can just call this `Batch`, e.g. `base.Batch` instead of `base.BaseBatch`.